src/android/Capture.java

```diff
@@ -269,6 +264,17 @@ private boolean isMissingCameraPermissions(Request req, String mediaPermission)
         return isMissingPermissions(req, cameraPermissions);
     }
 
+    private String getTempDirectoryPath() {
+        File cache = null;
+
+        // Use internal storage
+        cache = cordova.getActivity().getCacheDir();
+
+        // Create the cache directory if it doesn't exist
+        cache.mkdirs();
+        return cache.getAbsolutePath();
+    }
```

Comment:
This will need to be updated to match the file path declared in the providers xml list. See comments on `mediacapture_provider_paths.xml`